src/presenter.rs

```diff
@@ -81,21 +81,63 @@ impl Presenter {
             self.print_vulnerability(vulnerability, &tree);
         }
 
+        let deny_unmaintained = self
```

Comment:
Parsing `report.warnings` here instead of storing warnings by type in `Report` struct in `rustsec` crate